README.md

```diff
@@ -32,18 +57,50 @@ Sumo Logic Kubernetes Collection Helm Operator for the [Sumo Logic Kubernetes Co
   kubeclt apply -f config/samples/default_openshift.yaml
   ```
 
-## Configuration
-
-Configuration for Sumo Logic Kubernetes Collection Helm Operator needs to be provided in `SumologicCollection` resource.
-Custom Resource Definition for `SumologicCollection` is defined in
-[helm-operator.sumologic.com_sumologiccollections.yaml][crd].
-
-`SumologicCollection.spec` allows to configure all parameters from [values.yaml][values.yaml] for the
-[Sumo Logic Kubernetes Collection Helm Chart][helm-chart-repo].
-All possible parameters with descriptions can be found in [Configuration][helm-chart-configuration]
-section for Sumo Logic Kubernetes Collection Helm Chart.
-
-Example configurations for Sumo Logic Kubernetes Collection Helm Operator are available in [config/samples](config/samples) directory.
+### Verification
+
+Verify that Pod related to Sumologic Kubernetes Collection are running,
+for default configuration you should see similar list of Pods:
+
+```bash
+$ kubectl get pods -n openshift-operators
```

Comment:
Is it really in the `openshift-operators` namespace?

Reply:
Yes, this namespace is used to set operator scope to the whole cluster (and this is selected by default). It is possible to deploy operator in different namespace but then scope of this operator is limited only to this namespace.